src/customers/domain/customer.domain.ts

```diff
@@ -1,6 +1,11 @@
 import Order from '../../orders/domain/order.domain';
 import { Column, Entity, OneToMany, PrimaryGeneratedColumn } from 'typeorm';
 
+export enum CustomerRole {
```

Comment:
You can add this enum to its own file. It's better to separate the classes/enums. One class/entity per file.

Reply:
done